project/demos/arrive/Arrive.tscn

```diff
@@ -0,0 +1,25 @@
+[gd_scene load_steps=6 format=2]
+
+[ext_resource path="res://demos/arrive/Arriver.gd" type="Script" id=1]
+[ext_resource path="res://demos/arrive/ArriveDemo.gd" type="Script" id=2]
```

Comment:
Let's keep the name of the attached script & the scene files the same, it's easier to follow. `ArriveDemo.tscn` -> `ArriveDemo.gd`